chart/elastalert2/README.md

```diff
@@ -56,6 +56,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `command`                                    | command override for container                                                                                                | `NULL`                                                   |
 | `args`                                       | args override for container                                                                                                   | `NULL`                                                   |
 | `replicaCount`                               | number of replicas to run                                                                                                     | 1                                                        |
+| `rulesFolder`                                | Locaton of rules directory. Usefull when you have one docker image and different set on rules per environemnt. For example development can reside in `/opt/elastalert/develop` and production in `/opt/elastalert/production`.                                                                                                     | /opt/elastalert/rules                                                       |
```

Comment:
Can you help me understand this a bit better, in case others ask about it in the future? Typically when I am deploying a different set of rules to alternate environments my override yaml file defines the rules specific to each environment. The `jertel/elastalert2` Docker image remains the same across all environments. How does changing this directory name make a difference?

Reply:
Of course.
As far as I know there are few ways to have this implemented right now.
We can (for example) define our own secret and mount it in the Pod.

But what we wanted to have is to have as little copy-pasting and being able to new rules locally with docker-compose.
So, we have prepared set of yml files, which contain rules definitions. They are different per environment.
During local development, devs just add files to appropriate directory and run docker-compose.
![image](https://user-images.githubusercontent.com/19409072/164008267-6b233913-4b7e-454a-8a15-c572d7f4c317.png)

Our docker image just takes all those rules (both lab and prod) and mounts them in container (both set of rules are present in the container). And now I would like to be able, to tell ElastAlert that I want to use rules from lets say `lab` subdirectory.

Reply:
Thanks for that detailed explanation, and thanks for the contribution!